tests/unit_tests/test_analytics/sentry_relay.py

```diff
@@ -0,0 +1,139 @@
+import gzip
+import socket
+import threading
+import time
+from typing import Any, Dict, Union
+
+import flask
+import requests
+from flask import request
+from sentry_sdk.envelope import Envelope
+
+
+class SentryResponse:
+    def __init__(
+        self,
+        message,
+        project_id,
+        public_key,
+        tags: Union[None, Dict[str, Any]] = None,
+        is_error=False,
```

Comment:
why only tags are typed? haha

Reply:
also you could use:
`from __future__ import annotations` and use the newest typing style